warehouse/query-core/src/main/java/datawave/query/planner/DefaultQueryPlanner.java

```diff
@@ -746,7 +746,8 @@ protected ASTJexlScript updateQueryTree(ScannerFactory scannerFactory, MetadataH
         
         stopwatch.stop();
         
-        if (enforceUniqueTermsWithinExpressions) {
+        // Enforce if either DefaultQueryPlanner or ShardQueryConfig is true
+        if (enforceUniqueTermsWithinExpressions || config.getEnforceUniqueTermsWithinExpressions()) {
```

Comment:
Why does this term exist in two places?  Why not only have it in the shard query configuration?

Reply:
My understanding is that the LookupUUIDTune is used to override a select number of query configs, most of the options here are duplicates.